StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkPaymentController.swift

```diff
@@ -147,7 +147,8 @@ import UIKit
                                                    paymentMethodType: .link,
                                                    customerName: configuration.defaultBillingDetails.name,
                                                    customerEmailAddress: configuration.defaultBillingDetails.email,
-                                                   additionalParameteres: parameters) { [weak self] linkAccountSession, error in
+                                                   linkMode: .linkPaymentMethod,
```

Comment:
@carlosmuvi-stripe Do you think it’s fine to pass this for the custom integration?